README.rst

```diff
@@ -23,20 +23,31 @@ Building is as simple as
 ::
 
         $ git clone https://github.com/FreeRCT/FreeRCT.git
-        $ cd FreeRCT # Go into the downloaded source directory.
-        $ cmake .    # Checks libraries are where they're supposed to be and replaces some strings
-        $ make       # Let make do the heavy work.
+        $ cd FreeRCT    # Go into the downloaded source directory.
+        $ cmake .       # Checks libraries are where they're supposed to be and replaces some strings.
+        $ make          # Let make do the heavy work.
+        $ make install  # Optional step to install the program on the system. May require root privileges.
 
 
 CMake accepts the following options:
 
-================= ======= ==========================================================================
-Name              Default Explanation
-================= ======= ==========================================================================
-OVERRIDE_VCS      OFF     Override the lack of a VCS checkout.
-ASAN              OFF     Use AddressSanitizer (https://clang.llvm.org/docs/AddressSanitizer.html).
-USERDATA_PREFIX   '.'     Directory where user data such as savegames is stored.
-================= ======= ==========================================================================
+======================= ============================= ================================================================================
+Name                    Default value                 Explanation
+======================= ============================= ================================================================================
+OVERRIDE_VCS            OFF                           Override the lack of a VCS checkout. Supported VCSs are git and bzr.
+VERSION_STRING          '0.0.0-norev'                 If OVERRIDE_VCS is enabled, use this string as the build version.
+ASAN                    OFF                           Use AddressSanitizer (https://clang.llvm.org/docs/AddressSanitizer.html).
+CMAKE_INSTALL_PREFIX    '/usr'                        Directory where 'make install' installes 
+USERDATA_PREFIX         '~/.freerct' (Windows) or     Directory where user data such as savegames is stored.
+                        '~/.config/freerct' (other)
```

Comment:
You may want to make a note below saying that `~` is expanded by FreeRCT itself, to avoid packagers believing the documentation is silently assuming a user-only install.

(Packagers want a runtime expansion of `~` and we provide that, but without telling here what `~` actually does here, any Unix packager will assume the example uses shell expansion at  build time.)

Reply:
done